index.html

```diff
@@ -13,18 +13,18 @@
       </ul>
     </div>
     <script>    
-      var ul = document.getElementById('js-list');
+      const ul = document.getElementById('js-list');
       for(var i=1; i<=2; i++) {
```

Comment:
[IMO]
Starting with ES6, let is recommended instead of ver

Reply:
Thank you for your suggestions! I checked and changed it.